gebru/roster.json

```diff
@@ -126,6 +126,10 @@
     {
       "name": "Dan Willis",
       "github": "WalkingZ3d"
+      "materials": [
+        {
+            "type": "cv"
+        }]
```

Comment:
Can you please add a comma between "github": "WalkingZ3d" and "materials": [ ?

Reply:
I have edited it on github, did this work?